src/javascript-node-mongo/devcontainer-template.json

```diff
@@ -11,14 +11,12 @@
             "type": "string",
             "description": "Node.js version (use -bullseye variants on local arm64/Apple Silicon):",
             "proposals": [
```

Comment:
From https://www.mongodb.com/docs/database-tools/installation/installation/#platform-support & https://github.com/devcontainers/templates/actions/runs/6564315875/job/17830523847?pr=213#step:6:396

mongoDB tools doesn't have support for debian 12 (bookworm).